requirements.txt

```diff
@@ -15,3 +15,4 @@ lmdb==0.87
 h5py>=2.2.1,<=2.6.0
 pydot==1.0.28
 psutil>=1.2.1,<=3.4.2
+matplotlib==1.5.1
```

Comment:
I was finally able to get matplotlib 1.3.1 and verify that the code is working. Phew!